docs/richtext-commonmark.md

```diff
@@ -16,31 +16,40 @@ dependencies {
 The simplest way to render markdown is just pass a string to the [`Markdown`](/api/com.zachklipp.richtext.markdown/-markdown/)
 composable:
 
-```kotlin
+~~~kotlin
 Markdown(
   """
-    # Markdown
-
-    Library for rendering Markdown in Compose using Atlassian's [CommonMark](https://github.com/atlassian/commonmark-java)
-    library to parse, and `richtext-ui` to render.
-
-    ## Gradle
-
-    ```groovy
-    dependencies {
-      implementation "com.zachklipp.compose-richtext:richtext-commonmark:{richtext_version}"
-    }
-    ```
-
-    ## Usage
-
-    ```kotlin
-    // etc.
-    ```
+  # Demo
+  
+  Emphasis, aka italics, with *asterisks* or _underscores_. Strong emphasis, aka bold, with **asterisks** or __underscores__. Combined emphasis with **asterisks and _underscores_**. [Links with two blocks, text in square-brackets, destination is in parentheses.](https://www.example.com). Inline `code` has `back-ticks around` it.
+  
+  1. First ordered list item
+  2. Another item
+      * Unordered sub-list.
+  3. And another item.
+      You can have properly indented paragraphs within list items. Notice the blank line above, and the leading spaces (at least one, but we'll use three here to also align the raw Markdown).
+
+  * Unordered list can use asterisks
+  - Or minuses
+  + Or pluses
+  ---
+
+  ```javascript
```

Comment:
Why not a Kotlin example? 😋

Reply:
We don’t currently support syntax highlighting, so it doesn’t make too much of a difference. But I’d like to make that possible. I think this is something that should probably be solved at the markdown level, eg maybe an additional parameter on the ‘Markdown’ composable that is a composable function that takes the some kind of code block ast abstraction and can render it however it wants.